api/apiResponse.go

```diff
@@ -0,0 +1,33 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+type Response struct {
```

Comment:
can modify this to have few more field

type V2Resp struct {
	RequestID        string      `json:"request_id"`
	ErrorCode        int         `json:"error_code"`
	ErrorDescription string      `json:"error_description"`
	Data             interface{} `json:"data"`
}